Modules/IO/ImageBase/test/CMakeLists.txt

```diff
@@ -933,7 +933,8 @@ itk_add_test(
   COMMAND
   itkUnicodeIOTest)
 
-set(ITKIOImageBaseGTests itkWriteImageFunctionGTest.cxx)
```

Comment:
Perhaps change commit message to say "ENH: Add a test which exposes a bug in ImageIO with ReverseOrder"

Reply:
When I wrote the test, I didn't realize it was only a TIFFIO bug. I'm going to move the test over to the TIFF directory too.